eng/pipelines/templates/steps/npm-release-task.yml

```diff
@@ -47,3 +46,11 @@ steps:
         arguments: -pathToArtifacts ${{parameters.PathToArtifacts}} -accessLevel "public" -tag "${{parameters.Tag}}" -additionalTag "${{parameters.AdditionalTag}}" -registry ${{parameters.Registry}} -npmToken $(azure-sdk-npm-token)
         pwsh: true
       condition: succeeded()
+
+  - task: Powershell@2
+    displayName: Verify package tags
+    inputs:
+      targetType: filePath
+      filePath: eng/scripts/verify-npm-tags.ps1
+      arguments: -packageName ${{parameters.Artifact.name}} -workingDirectory $(System.DefaultWorkingDirectory)
```

Comment:
The publish steps are taking a PathToArtifacts and publishing whatever is in there. Why would this be different? In fact I'm not even sure Artifact parameter to this template is useful any longer. While it may be currently true that we only ever have one artifact in the JS templates right no I'm trying to ensure what we have will translate to our other publishing scenarios. For example, the partner release pipeline people can upload any number of packages into a folder and we publish them all.